src/error.rs

```diff
@@ -60,10 +62,34 @@ impl std::error::Error for Error {
     fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
         match self {
             Error::CommandIoError { source, .. } => Some(&**source),
-            Error::NoArgumentsGiven
-            | Error::NonZeroExitCode { .. }
-            | Error::InvalidUtf8ToStdout { .. }
-            | Error::InvalidUtf8ToStderr { .. } => None,
+            Error::InvalidUtf8ToStdout { source, .. }
+            | Error::InvalidUtf8ToStderr { source, .. } => Some(&**source),
```

Comment:
Is it not possible to produce a `&dyn` reference without an arc?

Reply:
Good catch. I thought I needed the `Arc` because `cradle::Error` implements `Clone`. But `FromUtf8Error` does too.

Reply:
Never mind. On older rust versions `FromUtf8Error` does *not* implement `Clone`. So I'll put the `Arc` back in.